cfn/README-GD.md

```diff
@@ -59,125 +67,110 @@ An example of a successful response is:
 }
 ```
 
-**Important:** If the command returns an error about not having the necessary role, please verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
-
 Make a note of the `access_key_id` and `secret_key` values, which you need to deploy the CloudFormation template to your AWS account. 
 
-**Note:** Each user can create only five access keys. If a "limit exceeded" response appears, you must delete one or more access keys before you can create new keys. 
-
-**Type the following command to list access keys:**
-```
-curl -s -X GET -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys | jq
-```
-
-**Use the selected access_key_id in the following command to delete the key:**
-
-```
-curl -X DELETE -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys/<ACCESS_KEY_ID_HERE>
-```
+**Necessary role error**
 
+If the command returns an error about not having the necessary role, please verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
 
-### Windows
+**"Limit exceeded" error**
 
-Check the version of PowerShell on your system by running PowerShell and typing in:
+Each user can create only five access keys. If a "limit exceeded" response appears, you must delete one or more access keys before you can create new keys. 
 
-```
-PS C:\Users\Username> $PSVersionTable.PSVersion
-```
+1. Type the following command to list access keys:
 
-An example of output:
+  ```
```

Comment:
Please reformat it with more spaces, so ordered list items would show it as 1 and 2 item:

1. Type the following command to list access keys:
    ```
    curl -s -X GET -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys | jq
    ```
1. Use the selected access_key_id in the following command to delete the key:
    ```
    curl -X DELETE -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys/<ACCESS_KEY_ID_HERE>
    ```

(you can just copy that from my comment ^^)

Currently it is show as `1.` and `1.`:

https://github.com/Shaddakim/cwe-collector/blob/05ff7573bd5640200014e7d848c1eebf03b998f6/cfn/README-GD.md
![image](https://user-images.githubusercontent.com/3668455/35329598-61d0e596-00f8-11e8-86ff-f7cdfc7f8186.png)